main/acle.md

```diff
@@ -5516,6 +5578,56 @@ each architecture includes its predecessor instruction set.
 | WFI             |           | 8,6K, 6-M     | `__wfi`                                               |
 | YIELD           |           | 8,6K, 6-M     | `__yield`                                             |
 
+# About FP8 intrinsics
+
+The specification for FP8 intrinsics is in
+[**Alpha** state](#current-status-and-anticipated-changes).
+
+Each 8-bit floating point intrinsic call has a parameter to define the format,
+saturation, and scale of its operands. This parameter is typically
+declared as `fpm_t fpm`.
+
+```c
+  typedef uint64_t fpm_t;
+```
+
+The ACLE declares several helper types and functions to
+facilitate construction of `fpm` arguments.
+
+## Support enumerations
+
+```c
+enum __ARM_FPM_FORMAT {
+  __ARM_FPM_E5M2,
+  __ARM_FPM_E4M3,
+};
+
+enum __ARM_FPM_OVERFLOW {
+  __ARM_FPM_INFNAN,
+  __ARM_FPM_SATURATE,
+};
+```
+
+## Helper functions
+
+```c
+  fpm_t arm_fpm_init();
+
+  fpm_t arm_set_fpm_src1_format(fpm_t fpm, enum __ARM_FPM_FORMAT format);
+  fpm_t arm_set_fpm_src2_format(fpm_t fpm, enum __ARM_FPM_FORMAT format);
+  fpm_t arm_set_fpm_dst_format(fpm_t fpm,  enum __ARM_FPM_FORMAT format);
+
+  fpm_t arm_set_fpm_overflow_cvt(fpm_t fpm, enum __ARM_FPM_OVERFLOW behaviour);
+  fpm_t arm_set_fpm_overflow_mul(fpm_t fpm, enum __ARM_FPM_OVERFLOW behaviour);
+
+  fpm_t arm_set_fpm_lscale(fpm_t fpm, uint64_t scale);
+  fpm_t arm_set_fpm_lscale2(fpm_t fpm, uint64_t scale);
+  fpm_t arm_set_fpm_nscale(fpm_t fpm,  int64_t scale);
```

Comment:
How should scale be interpreted? we have a specific number of bits, what should the behaviour be if the value provided is higher than what can fit in them?